stdlib/io-console/0/io-console.rbs

```diff
@@ -0,0 +1,137 @@
+class IO
+  class ConsoleMode
+    def echo=: ((true | false)) -> (true | false)
```

Comment:
How about using `bool` instead of `true | false`?
https://github.com/ruby/rbs/blob/edfa19dc7517b0d87d44a144ffcdbc40e4883fde/docs/syntax.md#bool-or-boolish

Reply:
Sider of CI say "Prefer boolish over bool for method arguments and block return values"

https://sider.review/gh/repos/174796313/pulls/783#issue-6033330

But here is not intended to boolish.

Reply:
Ah, thanks for telling the Sider issue.

I think we can ignore the Sier issue and can use `bool` instead. Because here is not intended to `boolish` as you said.
We don't have to follow Sider issues when the issue has a justification that matches the intention. For instance, in this case, the Sider issue says "When you strictly want true | false." as a justification and we want strictly `true | false`.

So let leave the Sider issue and tell us the issue looks a false positive.


By the way, I think we should consider improving something to avoid the confusion. For example, we can update the justification and CONTRIBUTING.md. I'll try it :muscle:

Reply:
OK Thanks!
I'll use `bool` instead.

I'll help if there's anything I can do!